openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsSlider/SuggestionsSlider.tsx

```diff
@@ -45,7 +45,7 @@ const SuggestionsSlider = () => {
           data-testid="more-suggestion-button"
           loading={loading}
           type="primary"
-          onClick={fetchSuggestions}>
+          onClick={() => fetchSuggestions()}>
```

Comment:
What we fixes here? Like there were params that causing issue?

Reply:
Yes, so we have one argument in the  `fetchSuggestions` which is optional. And since here it is directly aligned with the onClick so the onClick reference is going as the param which was causing the issue.